File: FragranciesSpringProject/src/main/java/com/miluna/FraganciesProject/dto/Fragrance.java
package com.miluna.FraganciesProject.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;


@AllArgsConstructor
@NoArgsConstructor
@Data
@Document(collection = "fragrances")
public class Fragrance {

    @Id
    String id;

    Long number;

    String name;

}

File: FragranciesFront/fragrances/src/components/Fragrance.js
import React from 'react';
import ListItem from '@material-ui/core/ListItem';
import ListItemText from '@material-ui/core/ListItemText';

const fragrance = ({data}) => {
    return (
        <ListItem>
            <ListItemText primary={data.number + " - " + data.name}/>
        </ListItem>
    )
};

export default fragrance;
